src/server/index.ts

```diff
@@ -106,5 +106,8 @@ async function launchServer(): Promise<void> {
 	);
 }
 
+// for testing
+export default app;
```

Comment:
Why is this needed?

Reply:
For future tests, may need to set up app